apps/nextra/next-env.d.ts

```diff
@@ -1,5 +0,0 @@
-/// <reference types="next" />
-/// <reference types="next/image-types/global" />
-
-// NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
I dont think we should be deleting this one: https://nextjs.org/docs/pages/building-your-application/configuring/typescript#custom-type-declarations